admin/site-admin/external-services.md

```diff
@@ -0,0 +1,40 @@
+## Integrate external services
+
+Stoplight allows you to integrate external services, such as remote code repositories, into your Stoplight organization. Stoplight supports the following integrations:
+
+- Bitbucket
+- More coming soon <!-- QUERY: which services are we going to support? -->
+
+Access **External Services** from the Stoplight dashboard by selecting:
+
+**admin** > **Site Admin** > **External Services**
+
+## Add a new service
+
+When you add a new service, project files automatically sync.
```

Comment:
I think we need to be specific here for BitBucket as the mapping from their concepts to ours is somewhat unique. 

A BitBucket Project --> a Stoplight Organization 
A BitBucket Repo --> A Stoplight Project 

It will be better aligned for GitHub and GitLab, but we're likely going to need to call out each flavor of VCS connection uniquely.